src/app/reference/ci-workflows/upload-strategies/page.md

```diff
@@ -147,75 +159,107 @@ const config: PlaywrightTestConfig = {
     },
   ],
 };
+
 export default config;
 ```
 
 {% /tab %}
 {% /tabs %}
 
-## Upload some passing runs
+## Upload only for the primary branch
```

Comment:
I'm not sure about the wording for this section. Advice welcome.

Reply:
i think primary/main is fine - everybody should understand what that means (and that it might be `master` in their situation)